cloud-etl/docs/index.rst

```diff
@@ -0,0 +1,254 @@
+|                       |                       | <ksql.commands>`__) |
++-----------------------+-----------------------+---------------------+
+| GCS/S3/Blob sink      | KSQL tables           | GCS/S3/Blob         |
+| connector             | ``COUNT_PER_SOURCE``, |                     |
+|                       | ``SUM_PER_SOURCE``    |                     |
++-----------------------+-----------------------+---------------------+
+
+Warning
+=======
+
+This demo uses real cloud resources, including that of |ccloud|,
+Amazon Kinesis, and one of the cloud storage providers. To avoid
+unexpected charges, carefully evaluate the cost of resources before
+launching the demo and ensure all resources are destroyed after you are
+done running it.
+
+Prerequisites
+=============
+
+Cloud services
+--------------
+
+-  `Confluent Cloud cluster <https://confluent.cloud>`__: for development only. Do not use a production cluster.
+-  `Confluent Cloud KSQL <https://docs.confluent.io/current/quickstart/cloud-quickstart/ksql.html>`__ provisioned in your |ccloud|
+-  Download `Confluent Platform <https://www.confluent.io/download/>`__ |release|: for more advanced Confluent CLI functionality (optional)
+-  AWS or GCP or Azure access
+
+Local install
+-------------
+
+-  `Confluent Cloud CLI <https://docs.confluent.io/current/quickstart/cloud-quickstart/index.html#step-2-install-the-ccloud-cli>`__
+   v0.234.0 or later
+-  ``aws`` CLI, properly initialized with your credentials: AWS Kinesis and (optional) if destination is AWS S3
+-  ``gsutils`` CLI, properly initialized with your credentials: (optional) if destination is GPC GCS
+-  ``az`` CLI, properly initialized with your credentials: (optional) if destination is Azure Blob storage
+-  ``jq``
+-  ``curl``
+
+Run the Demo
+============
+
+Setup
+-----
+
+#. By default, the demo reads the configuration parameters for your |ccloud| cluster from a file at ``$HOME/.ccloud/config``. You can change this filename via the parameter ``CONFIG_FILE`` in :devx-examples:`config/demo.cfg|cloud-etl/config/demo.cfg`. Enter the configuration parameters for your |ccloud| cluster:
+
+   .. code:: shell
+
+      $ cat $HOME/.ccloud/config
+      bootstrap.servers=<BROKER ENDPOINT>
+      ssl.endpoint.identification.algorithm=https
+      security.protocol=SASL_SSL
+      sasl.mechanism=PLAIN
+      sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username\="<API KEY>" password\="<API SECRET>";
+      schema.registry.url=https://<SR ENDPOINT>
+      schema.registry.basic.auth.user.info=<SR API KEY>:<SR API SECRET>
+      basic.auth.credentials.source=USER_INFO
+      ksql.endpoint=https://<KSQL ENDPOINT>
+      ksql.basic.auth.user.info=<KSQL API KEY>:<KSQL API SECRET>
+
+   To get the right values for the endpoints in the file above, find them either via the |ccloud| UI or |ccloud| CLI commands. If you have multiple |ccloud| clusters, make sure to use the one with the associated KSQL cluster.
```

Comment:
Should we highlight for the user to create and fill in API keys and secrets here as well (or is that coming later somewhere else)?

Reply:
👍 excellent point, added.